resources/acsl/graphtheory.md

```diff
@@ -227,4 +227,63 @@ conclusions can be made for the rest of the numbers.
 
 # Sample Problems
 
+## 1. How many different cycles are contained in the directed graph visualized below:
 
+<img src="/res/acsl/graphtheory/prob1.png" width="250"/>
+
+One cycle is *ADA*, or *DAD*. Another is *ABCA*, which is also *BCAB* or *CABC*. The last cycle is *ECE*, or *CEC*.
+So, in total, there are 3 different cycles in the graph.
+
+## 2. Using the adjacency matrix below, draw the directed graph.
+
+| `M` | A | B | C | D | E |
+| --- | --- | --- | --- | --- | --- |
+| A | 1 | 0 | 1 | 1 | 0 |
+| B | 0 | 1 | 0 | 1 | 1 |
+| C | 0 | 0 | 0 | 1 | 0 |
+| D | 0 | 1 | 1 | 0 | 1 |
+| E | 1 | 0 | 1 | 1 | 1 |
+
+The drawn graph below is only one way to depict this matrix.
+
+| Step | Graph <img width="300"/> | Step | Graph <img width="300"/>|
```

Comment:
you can drop the img widths (unless for some reason this makes formatting wonky... in which case show me)

Reply:
Noted, edited it out and committed the changes